Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 45, in <module>
    folder = all_files[num_file]
             ~~~~~~~~~^^^^^^^^^^
IndexError: list index out of range
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'variable (17'
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'variable (17'
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                                                   ~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'variable (17'
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                                                   ~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                                                   ~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'variable (17'
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'variable (17'
score/main.py", line 82, in <module>
    io.write(nwbfile)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/backends/hdf5/h5tools.py", line 375, in write
    super().write(**kwargs)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/backends/io.py", line 81, in write
    f_builder = self.__manager.build(container, source=self.__source, root=True)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 171, in build
    result = self.__type_map.build(container, self, source=source, spec_ext=spec_ext, export=export)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 770, in build
    builder = obj_mapper.build(container, manager, builder=builder, source=source, spec_ext=spec_ext, export=export)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 714, in build
    self.__add_groups(builder, self.__spec.groups, container, manager, source, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1035, in __add_groups
    self.__add_groups(sub_builder, spec.groups, container, build_manager, source, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1035, in __add_groups
    self.__add_groups(sub_builder, spec.groups, container, build_manager, source, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1048, in __add_groups
    self.__add_containers(builder, spec, attr_value, build_manager, source, container, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1068, in __add_containers
    new_builder = build_manager.build(value, source=source, spec_ext=spec, export=export)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 171, in build
    result = self.__type_map.build(container, self, source=source, spec_ext=spec_ext, export=export)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 770, in build
    builder = obj_mapper.build(container, manager, builder=builder, source=source, spec_ext=spec_ext, export=export)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 713, in build
    self.__add_datasets(builder, self.__spec.datasets, container, manager, source, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1018, in __add_datasets
    self.__add_containers(builder, spec, attr_value, build_manager, source, container, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1106, in __add_containers
    self.__add_containers(builder, spec, container, build_manager, source, parent_container, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1068, in __add_containers
    new_builder = build_manager.build(value, source=source, spec_ext=spec, export=export)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 171, in build
    result = self.__type_map.build(container, self, source=source, spec_ext=spec_ext, export=export)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 770, in build
    builder = obj_mapper.build(container, manager, builder=builder, source=source, spec_ext=spec_ext, export=export)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 755, in build
    raise Exception(msg) from ex
Exception: could not resolve dtype for VectorData 'location'
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'variable (17'
Traceback (most recent call last):
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 752, in build
    bldr_data, dtype = self.convert_dtype(spec, container.data, spec_dtype=spec_dtype)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 200, in convert_dtype
    ret, ret_dtype = cls.__check_edgecases(spec, value, spec_dtype)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 313, in __check_edgecases
    raise ValueError(msg)
ValueError: Cannot infer dtype of empty list or tuple. Please use numpy array with specified dtype.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 82, in <module>
    io.write(nwbfile)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/backends/hdf5/h5tools.py", line 375, in write
    super().write(**kwargs)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/backends/io.py", line 81, in write
    f_builder = self.__manager.build(container, source=self.__source, root=True)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 171, in build
    result = self.__type_map.build(container, self, source=source, spec_ext=spec_ext, export=export)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 770, in build
    builder = obj_mapper.build(container, manager, builder=builder, source=source, spec_ext=spec_ext, export=export)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 714, in build
    self.__add_groups(builder, self.__spec.groups, container, manager, source, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1035, in __add_groups
    self.__add_groups(sub_builder, spec.groups, container, build_manager, source, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1035, in __add_groups
    self.__add_groups(sub_builder, spec.groups, container, build_manager, source, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1048, in __add_groups
    self.__add_containers(builder, spec, attr_value, build_manager, source, container, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1068, in __add_containers
    new_builder = build_manager.build(value, source=source, spec_ext=spec, export=export)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 171, in build
    result = self.__type_map.build(container, self, source=source, spec_ext=spec_ext, export=export)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 770, in build
    builder = obj_mapper.build(container, manager, builder=builder, source=source, spec_ext=spec_ext, export=export)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 713, in build
    self.__add_datasets(builder, self.__spec.datasets, container, manager, source, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1018, in __add_datasets
    self.__add_containers(builder, spec, attr_value, build_manager, source, container, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1106, in __add_containers
    self.__add_containers(builder, spec, container, build_manager, source, parent_container, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1068, in __add_containers
    new_builder = build_manager.build(value, source=source, spec_ext=spec, export=export)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 171, in build
    result = self.__type_map.build(container, self, source=source, spec_ext=spec_ext, export=export)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 770, in build
    builder = obj_mapper.build(container, manager, builder=builder, source=source, spec_ext=spec_ext, export=export)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 755, in build
    raise Exception(msg) from ex
Exception: could not resolve dtype for VectorData 'location'
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                                                   ~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                                                   ~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'variable (17'
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 103, in create_nwb
    all_files = sorted(os.listdir(os.path.join(path, 'SpikeTimes')))
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/braintree/home/aliya277/dandi_brainscore/inventory/norm_FOSS.sub_pico.20230831_121100.proc/SpikeTimes'
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'variable (17'
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'variable (17'
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                                                   ~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
/var/slurm/slurmd/job32885245/slurm_script: line 17: 2531784 Killed                  python main.py $SLURM_ARRAY_TASK_ID ${*:1}
slurmstepd: error: Detected 1 oom-kill event(s) in StepId=32885245.batch. Some of your processes may have been killed by the cgroup out-of-memory handler.
Traceback (most recent call last):
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 752, in build
    bldr_data, dtype = self.convert_dtype(spec, container.data, spec_dtype=spec_dtype)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 200, in convert_dtype
    ret, ret_dtype = cls.__check_edgecases(spec, value, spec_dtype)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 313, in __check_edgecases
    raise ValueError(msg)
ValueError: Cannot infer dtype of empty list or tuple. Please use numpy array with specified dtype.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 82, in <module>
    io.write(nwbfile)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/backends/hdf5/h5tools.py", line 375, in write
    super().write(**kwargs)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/backends/io.py", line 81, in write
    f_builder = self.__manager.build(container, source=self.__source, root=True)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 171, in build
    result = self.__type_map.build(container, self, source=source, spec_ext=spec_ext, export=export)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 770, in build
    builder = obj_mapper.build(container, manager, builder=builder, source=source, spec_ext=spec_ext, export=export)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 714, in build
    self.__add_groups(builder, self.__spec.groups, container, manager, source, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1035, in __add_groups
    self.__add_groups(sub_builder, spec.groups, container, build_manager, source, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1035, in __add_groups
    self.__add_groups(sub_builder, spec.groups, container, build_manager, source, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1048, in __add_groups
    self.__add_containers(builder, spec, attr_value, build_manager, source, container, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1068, in __add_containers
    new_builder = build_manager.build(value, source=source, spec_ext=spec, export=export)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 171, in build
    result = self.__type_map.build(container, self, source=source, spec_ext=spec_ext, export=export)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 770, in build
    builder = obj_mapper.build(container, manager, builder=builder, source=source, spec_ext=spec_ext, export=export)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 713, in build
    self.__add_datasets(builder, self.__spec.datasets, container, manager, source, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1018, in __add_datasets
    self.__add_containers(builder, spec, attr_value, build_manager, source, container, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1106, in __add_containers
    self.__add_containers(builder, spec, container, build_manager, source, parent_container, export)
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 1068, in __add_containers
    new_builder = build_manager.build(value, source=source, spec_ext=spec, export=export)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 171, in build
    result = self.__type_map.build(container, self, source=source, spec_ext=spec_ext, export=export)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/manager.py", line 770, in build
    builder = obj_mapper.build(container, manager, builder=builder, source=source, spec_ext=spec_ext, export=export)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/utils.py", line 644, in func_call
    return func(args[0], **pargs)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aliya277/anaconda3/lib/python3.11/site-packages/hdmf/build/objectmapper.py", line 755, in build
    raise Exception(msg) from ex
Exception: could not resolve dtype for VectorData 'location'
/var/slurm/slurmd/job32885244/slurm_script: line 17: 2531785 Killed                  python main.py $SLURM_ARRAY_TASK_ID ${*:1}
slurmstepd: error: Detected 1 oom-kill event(s) in StepId=32885244.batch. Some of your processes may have been killed by the cgroup out-of-memory handler.
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'variable (17'
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'variable (17'
Traceback (most recent call last):
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/main.py", line 54, in <module>
    nwbfile = create_nwb(config,path)
              ^^^^^^^^^^^^^^^^^^^^^^^
  File "/weka/scratch/weka/dicarlo/aliya277/dandi_brainscore/utils/nwb_helper.py", line 173, in create_nwb
    StimOnnOff = [float(line1.split('/')[0]),float(line1.split('/')[1])] 
                                                   ~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
/var/slurm/slurmd/job32885227/slurm_script: line 17: 4158740 Killed                  python main.py $SLURM_ARRAY_TASK_ID ${*:1}
slurmstepd: error: Detected 1 oom-kill event(s) in StepId=32885227.batch. Some of your processes may have been killed by the cgroup out-of-memory handler.
slurmstepd: error: *** JOB 32885230 ON node113 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885273 ON node041 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885270 ON node103 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885231 ON node107 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885253 ON node107 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885246 ON node114 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885252 ON node107 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885233 ON node107 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885251 ON node107 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885254 ON node107 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885260 ON node107 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885262 ON node107 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885259 ON node107 CANCELLED AT 2023-10-03T16:11:43 ***
slurmstepd: error: *** JOB 32885223 ON node100 CANCELLED AT 2023-10-03T16:11:45 ***
slurmstepd: error: *** JOB 32885236 ON node091 CANCELLED AT 2023-10-03T16:13:41 ***
slurmstepd: error: *** JOB 32885235 ON node091 CANCELLED AT 2023-10-03T16:17:18 ***
slurmstepd: error: *** JOB 32885234 STEPD TERslurmstepd: error: *** JOB 32885239 STEPD TERMINATED ON node114 AT 2023-10-03T16:41:43 DUE TO JOB NOT ENDING WITH SIGNALS ***
MINATED ON node114 AT 2023-10-03T16:41:43 DUE TO JOB NOT ENDING WITH SIGNALS ***
MINATED ON node107 AT 2023-10-03T16:41:43 DUE TO JOB NOT ENDING WITH SIGNALS ***
slurmstepd: error: *** JOB 32885231 STEPD TERMINATED ON node107 AT 2023-10-03T16:41:43 DUE TO JOB NOT ENDING WITH SIGNALS ***
slurmstepd: error: *** JOB 32885270 STEPD TERMINATED ON node103 AT 2023-10-03T16:41:44 DUE TO JOB NOT ENDING WITH SIGNALS ***
slurmstepd: error: *** JOB 32885268 STEPD TERMINATED ON node103 AT 2023-10-03T16:41:44 DUE TO JOB NOT ENDING WITH SIGNALS ***
